contents/docs/contribute/contribute-to-website.md

```diff
@@ -0,0 +1,336 @@
+---
+title: "Contribute to the website: documentation, handbook, and blog"
+sidebar: Docs
+showTitle: true
+---
+
+You can contribute to the PostHog documentation, handbook, and blog in two ways:
+
+1. You can create a Pull Request in GitHub for any page that has an **Edit this page** link on it. In this situation you must edit the page using the GitHub web editor interface. This method is suitable for text-only edits and basic file manipulation, such as renaming.
+2. You can run the posthog.com website locally and make changes there by creating a branch of the master codebase, committing changes to that branch and raising a Pull Request to merge those changes. This is the recommended method as it allows you to quickly preview your changes, as well as perform more complex changes easily.
+
+Below, we'll explain how to set up option two. Most PostHog employees use Mac laptops in their day-to-day work, so this guide focuses on making changes via a Mac, rather than a Windows computer.
+
+> **Should I use the terminal or GitHub desktop?**<br/>
+This guide explains how to make changes either in the terminal, or via Github Desktop. If you're unfamiliar with the terminal or GitHub, we recommend using GitHub Desktop to make changes. You may also find [GitHub's glossary of terms](https://docs.github.com/en/get-started/quickstart/github-glossary) to be a helpful reference if you're entirely new to GitHub.
+
+## Editing posthog.com locally
+
+### Before you begin
+
+In order to run the PostHog website locally, you need to install the following, via the links below:
+
+- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
```

Comment:
Which git do I download (obvs mac version) as there are 5 mac versions to choose from ? We should probably make a suggestion or say all are fine (as I am unsure
![Screenshot 2021-10-01 at 09 52 46](https://user-images.githubusercontent.com/65415371/135593056-d385dda2-fb55-4c1b-8a98-41d4b5c0fb07.png)
)

Reply:
I went with homebrew which sounds a lil sketchy, like moonshine, but hey life is made of risks

Reply:
@lottiecoxon that's the way to go for a true Mac pro 🙌

Reply:
Seems that [GitHub Desktop also installs Git if it's not already installed](https://github.com/git-guides/install-git#:~:text=Install%20Git%20Using%20GitHub%20Desktop,don't%20already%20have%20it.&text=Regardless%20of%20if%20you%20have,simple%20collaboration%20tool%20for%20Git.) so we can remove that requirement.